src/zenml/client.py

```diff
@@ -2642,69 +2761,133 @@ def list_artifacts(
             user_id=user_id,
             only_unused=only_unused,
         )
-        artifact_filter_model.set_scope_workspace(self.active_workspace.id)
-        return self.zen_store.list_artifacts(artifact_filter_model)
+        artifact_version_filter_model.set_scope_workspace(
+            self.active_workspace.id
+        )
+        return self.zen_store.list_artifact_versions(
+            artifact_version_filter_model
+        )
 
-    def delete_artifact(
+    def update_artifact_version(
+        self,
+        name_id_or_prefix: Union[str, UUID],
+        version: Optional[str] = None,
+        add_tags: Optional[List[str]] = None,
+        remove_tags: Optional[List[str]] = None,
+    ) -> ArtifactVersionResponse:
+        """Update an artifact version.
+
+        Args:
+            name_id_or_prefix: The name, ID or prefix of the artifact to update.
+            version: The version of the artifact to update. Only used if
+                `name_id_or_prefix` is the name of the artifact. If not
+                specified, the latest version is updated.
+            add_tags: Tags to add to the artifact version.
+            remove_tags: Tags to remove from the artifact version.
+
+        Returns:
+            The updated artifact version.
+        """
+        artifact_version = self.get_artifact_version(
+            name_id_or_prefix=name_id_or_prefix,
+            version=version,
+        )
+        artifact_version_update = ArtifactVersionUpdate(
+            add_tags=add_tags, remove_tags=remove_tags
+        )
+        return self.zen_store.update_artifact_version(
+            artifact_version_id=artifact_version.id,
+            artifact_version_update=artifact_version_update,
+        )
+
+    def delete_artifact_version(
         self,
         name_id_or_prefix: Union[str, UUID],
         version: Optional[str] = None,
         delete_metadata: bool = True,
         delete_from_artifact_store: bool = False,
     ) -> None:
-        """Delete an artifact.
+        """Delete an artifact version.
 
         By default, this will delete only the metadata of the artifact from the
-        database, not the artifact itself.
+        database, not the physical artifact in the artifact store.
```

Comment:
Physical artifact? Sounds like not the right adjective to use.

Reply:
Yeah not sure how to call it - any suggestions? It should refer to the actual object stored in the artifact store, as opposed to the entity stored in our ZenML DB

Reply:
'actual'? or just what you say: 'the actual object stored in the artifact store' etc?